mnemonic/cardano-sl-mnemonic.cabal

```diff
@@ -0,0 +1,99 @@
+name:                cardano-sl-mnemonic
+version:             2.0.0
+synopsis:            TODO
+description:         See README
+homepage:            https://github.com/input-output-hk/cardano-sl/mnemonic/README.md
+author:              IOHK Engineering Team
+maintainer:          operations@iohk.io
+copyright:           2018 IOHK
+license:             MIT
+license-file:        LICENSE
+category:            Data
+build-type:          Simple
+extra-source-files:  README.md
+cabal-version:       >=1.10
+
+library
+  default-language:   Haskell2010
+  default-extensions: DeriveGeneric
+                      LambdaCase
+                      NoImplicitPrelude
+                      OverloadedStrings
+                      TupleSections
+                      TypeApplications
+                      ScopedTypeVariables
+
+  hs-source-dirs:     src
+
+  build-depends: base >=4.7 && <5
+               , QuickCheck
+               , aeson
+               , base58-bytestring
+               , basement
+               , bytestring
+               , cardano-crypto
+               , cardano-sl
+               , cardano-sl-binary
+               , cardano-sl-chain
+               , cardano-sl-core
+               , cardano-sl-crypto
+               , cardano-sl-infra
+               , cardano-sl-util
+               , containers
+               , cryptonite
+               , data-default
+               , formatting
+               , lens
+               , memory
+               , optparse-applicative
+               , safe
+               , swagger2
+               , text
+               , time
+               , universum
+
+  exposed-modules: Cardano.Mnemonic.BIP39
+                 , Cardano.Mnemonic.Util
```

Comment:
Let's avoid the `BIP39` denomination completely and stick with `Cardano.Mnemonic` only.